ext/native-decls/SendNuiMessage.md

```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
 * **jsonString**: 
 
 ## Return value
+
+## Examples
+> LUA Example:
+```lua
+local data = {
+    action = "displaySomething",
+    text = "Hello, World!"
+}
+
+local jsonStr = json.encode(data)
+
+local result = SendNUIMessage(jsonStr)
+
+if result then
+    print("Message sent to NUI.")
+else
+    print("Failed to send message to NUI.")
+end
+```
+> C# Example:
+```c#
+using CitizenFX.Core;
+using Newtonsoft.Json;
+
+public class YourScript : BaseScript
+{
+    public YourScript()
+    {
+        EventHandlers["YourEventName"] += new Action<string>(OnNuiMessageReceived);
+    }
+
+    private void OnNuiMessageReceived(string jsonString)
+    {
+        dynamic data = JsonConvert.DeserializeObject(jsonString);
+
+        if (data.action == "displaySomething")
+        {
+            string text = data.text;
+            Debug.WriteLine($"Received NUI Message: {text}");
+        }
+    }
+
+    private void SendNuiMessage()
+    {
+        dynamic data = new
+        {
+            action = "displaySomething",
+            text = "Hello from C#!"
+        };
+
+        string jsonStr = JsonConvert.SerializeObject(data);
+
+        Function.Call((Hash)0x7AE0589093A2E088, jsonStr); 
```

Comment:
Please refrain from doing stuff like this on example code 😅

You could use something such as:

```
TriggerEvent("chat:addMessage", new
{
    color = new[] {255, 0, 0},
    multiline = true,
    args = new[] {"Me",  jsonStr}
});
```

More examples [here](https://docs.fivem.net/docs/resources/chat/events/chat-addMessage/)